PublicationsRouter/api/v3/Retrieve.md

```diff
@@ -129,17 +126,15 @@ In order to tell the difference between (1) and (2), compare the following two l
             "packaging" : "https://pubrouter.jisc.ac.uk/FilesAndJATS"
         },
         {
-            "type" : "fulltext",
-            "format" : "application/pdf",
-            "url" : "https://pubrouter.jisc.ac.uk/api/v3/notification/123456789/content/publisher.pdf",
+            "type": "fulltext", 
+            "format": "application/pdf", 
+            "url": "https://externallink.ac.uk/123456789"
```

Comment:
I'd suggest external link: `"https://some_publisher_site.com/some_file_name.pdf"` as almost certainly won't be ".ac.uk"